### ISSUE TITLE
ARIA role values in IE11 are treated as case sensitive

### ISSUE THREAD

**Issue by stevefaulkner** [FreedomScientific/VFO-standards-support#7]
### Summary
JAWS does not recognize role="buTTon" as a 'push button' in IE11 although it is exposed as a 'push button' in MSAA

### Steps to reproduce:

1. Go to https://s.codepen.io/stevef/debug/YrLwwE
2. move focus to the first focusable element, observe what JAWS announces
3. move focus to the second focusable element, observe what JAWS announces

### Expected result
JAWS announces both as buttons

### Actual result

JAWS only announces the the second element as a button

### Example

https://s.codepen.io/stevef/debug/YrLwwE

#### JAWS version and build number

 JAWS 18.0.4104
 
#### Operating System and version

Windows 10 Home 64-bit

#### Browser and version:

IE 11

Related Firefox bug: [aria ROLE values are case sensitive](https://bugzilla.mozilla.org/show_bug.cgi?id=1407167)

**Comment by gezlemon** [comment]
I can confirm the issue. JAWS doesn't announce the role unless the case is correct in IE11 and Firefox 56 (but does with Chrome). The correct role is being exposed by the browsers, despite the case.

**Comment by stevefaulkner** [comment]
JAWS bug Filed:
Bug #97006 - ARIA role values in IE11 are treated as case sensitive

**Comment by ggordon-vfo** [comment]
This was fixed in the JAWS 2018 update in March of 2018: version 2018.1803....
